src/app/components/entityView/status/StatusView.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import StatusIcon from "../../cells/status/StatusIcon";
+import f from "lodash/fp";
+
+const StatusView = props => {
+  const {
+    cell: { value, column },
+    langtag
+  } = props;
+
+  const valuesToRender = f.compose(
+    f.map(val => (
+      <StatusIcon
+        key={"StatusIcon " + val.name}
+        icon={val.icon}
+        tooltip={val.tooltip}
+        color={val.color}
+        displayName={val.displayName}
+        langtag={langtag}
+        blockMode
+      />
+    )),
+    f.filter({ value: true }),
+    f.zipWith((a, b) => ({ value: a, ...b }), value)
```

Comment:
[minor] Bitte explizitere Variablennamen, so ist die Funktion schwer nachvollziehbar. Sehe ich das richtig, dass du die hier in die Column-Rule die Werte aus `value` der Reihe nach einsetzt, um so Anzeigestrings zu bekommen?